sbt-dotty/sbt-test/sbt-dotty/quoted-example-project/build.sbt

```diff
@@ -0,0 +1,7 @@
+scalaVersion := sys.props("plugin.scalaVersion")
+
+libraryDependencies ++= Seq(
+      "ch.epfl.lamp" % "dotty_0.14" % scalaVersion.value,
+      "ch.epfl.lamp" % "dotty_0.14" % scalaVersion.value % "test->runtime",
```

Comment:
The dotty version should not be hardcoded

Reply:
Also use "dotty-compiler", not "dotty".

Reply:
Opened #6207 to for this issue